### ISSUE TITLE
TypeScript typings are missing in published bundle

### ISSUE THREAD

**Issue by n1ru4l** [feedback-fish/feedback-fish-react#9]
The published bundle points to type definitions existing at `dist/index.d.ts`, however, those are not included (https://unpkg.com/browse/@feedback-fish/react@1.2.0/package.json).
https://unpkg.com/browse/@feedback-fish/react@1.2.0/dist/

**Comment by n1ru4l** [comment]
Workaround for create-react-app:

Add `feedback-fish.d.ts` into `src`:


```tsx
import type { FunctionComponent } from "react";

/**
 * @source https://github.com/feedback-fish/feedback-fish-react/blob/f74cecea63903e0dffcdeb9cada45b2af8fd7f4f/src/FeedbackFish.tsx#L5-L14
 * TODO: fix this once https://github.com/feedback-fish/feedback-fish-react/issues/9 is resolved
 */
declare module "@feedback-fish/react" {
  type Props = {
    projectId: Parameters<typeof useFeedbackFish>[0];
    children?:
      | ReactElement
      | ((props: Record<string, unknown>) => ReactElement);
    userId?: string;
    metadata?: {
      [key: string]: string;
    };
  };

  export const FeedbackFish: FunctionComponent<Props>;
}
```

**Comment by mxstbr** [comment]
Fixed as of v1.2.1! Sorry for the delay :pray:
